includes/class-wc-stripe-status.php

```diff
@@ -194,4 +195,62 @@ public function render_status_report_section() {
 		</table>
 		<?php
 	}
+
+	/**
+	 * Add Stripe tools to the Woo debug tools.
+	 *
+	 * @param array $tools List of current available tools.
+	 */
+	public function debug_tools( $tools ) {
+		if ( WC_Stripe_Subscriptions_Helper::is_subscriptions_enabled() ) {
+			$tools['list_detached_subscriptions'] = [
+				'name'     => __( 'List Stripe subscriptions with detached payment method', 'woocommerce-gateway-stripe' ),
+				'button'   => __( 'List subscriptions', 'woocommerce-gateway-stripe' ),
+				'desc'     => sprintf(
+					'%1$s<br/><strong class="red">%2$s</strong> %3$s<br/><strong>%4$s</strong>',
+					__( 'This tool will list all Stripe subscriptions with detached payment methods.', 'woocommerce-gateway-stripe' ),
+					__( 'Note:', 'woocommerce-gateway-stripe' ),
+					__( 'This tool will make an API request to Stripe for each Stripe subscription in your store. For stores with many subscriptions, this may temporarily impact performance.', 'woocommerce-gateway-stripe' ),
```

Comment:
Should we refine this message a bit to make it clearer which subscriptions we're going to be checking?
```suggestion
					__( 'This tool will make an API request to Stripe for each active Stripe subscription in your store that is due to renew in the next month. For stores with many subscriptions, this may temporarily impact performance.', 'woocommerce-gateway-stripe' ),
```